### PR TITLE
JAVA-21305 : Changes made for adding ratpack module to jdk9-and above…

### PR DESCRIPTION
**Pull request by Bipinkumar27**
… module

### CODE REVIEW

File: web-modules/pom.xml [hunk 1]
```diff
@@ -25,12 +25,12 @@
         <module>javax-servlets-2</module>
         <module>jee-7</module>
         <module>jooby</module>
```
Comment (dkapil) [inline web-modules/pom.xml:27]: add a comment here , add the same comment against restx

File: web-modules/pom.xml [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
-        <module>ninja</module>-->
+        <module>linkrest</module>-->
+        <module>ninja</module>
         <!-- <module>play-modules</module> --> <!-- Not a maven project -->
         <!-- <module>raml-modules</module> --> <!-- Not a maven project -->
-        <!--<module>ratpack</module>
-        <module>resteasy</module>
+        <module>ratpack</module>
+        <!--<module>resteasy</module>
         <module>restx</module>
```
Comment (dkapil) [inline web-modules/pom.xml:34]: we need to define jooby and restx in the submodule fashion in the root pom.xml under *first and *second profile respectively so they are still executed in the jdk8 builds.

**Review by dkapil — CHANGES_REQUESTED** [review]

File: web-modules/pom.xml (hunk 1 above)
Comment (Bipinkumar27) [reply web-modules/pom.xml]: done

File: web-modules/pom.xml (hunk 2 above)
Comment (Bipinkumar27) [reply web-modules/pom.xml]: done

**Pull request merged**